storage/object/how-to/restore-an-object-from-glacier.mdx

```diff
@@ -40,10 +40,10 @@ categories:
     <Message type="tip">
       Alternatively, click <Icon name="more" /> > **Restore**.
     </Message>
-4. Enter the number of days after which the object will be transferred back to Glacier. 
+4. Enter the number of days after which the object will be transferred back to Glacier.
 5. Click **Restore object from S3 Glacier**.
     <Message type="note">
-      Your object remains available in `STANDARD` class for the duration you chose during the restore process. It will be transferred automatically back to `GLACIER` once the configured period is over. 
+      Your object remains available in `STANDARD` class for the duration you chose during the restore process. It will be transferred automatically back to `GLACIER` once the configured period is over.
```

Comment:
```suggestion
      Your object remains available in `STANDARD` class for the duration you chose during the restoring process. It will be transferred automatically back to `GLACIER` once the configured period is over.
```